python/cudf/cudf/dataframe/string.py

```diff
@@ -535,7 +535,11 @@ def element_indexing(self, arg):
             return columnops.as_column(out)
 
     def __getitem__(self, arg):
-        return self.element_indexing(arg)
+        col = self.element_indexing(arg)
+        if (not np.isscalar(arg)) and (np.ndim(col) == 0):
+            return columnops.as_column([col])
+        else:
+            return col
```

Comment:
This should be fixed in `element_indexing` instead of here.

Reply:
Updated the code to be handled in `element_indexing`